src/main/java/seedu/financialplanner/commands/OverviewCommand.java

```diff
@@ -20,44 +24,74 @@ public OverviewCommand(RawCommand rawCommand) {
 
     @Override
     public void execute() throws Exception {
-        CashflowList list = CashflowList.getInstance();
-        Ui.getInstance().showMessage("Here is an overview of your financials:\n" +
-                "Total balance: " + formatDoubleToString(Cashflow.getBalance()) + "\n" +
-                "Highest income: " + getHighestIncome(list) + "\n" +
-                "Highest expense: " + getHighestExpense(list) + "\n" +
-                "Remaining budget for the month: " + getBudgetDesc());
-        //todo: indicate description of income/expense
+        Ui.getInstance().printOverview(getBalance(), getHighestIncome(), getHighestExpense(),
```

Comment:
Wow, this line look very long. Maybe can separate out and store values in variables for readability

Reply:
changed!